### PR TITLE
[On-prem] Logging Stats

### PR DESCRIPTION
**Pull request by HaileyJang**
This PR prints naive total runtime and throughput GB per seconds of the on-prem CLI performance.

### CODE REVIEW

File: skyplane/cli/cli.py [hunk 1]
```diff
@@ -142,7 +144,20 @@ def cp(
...
+            start = time.perf_counter()
+            rc = os.system(cmd)
+            request_time = time.perf_counter() - start
+
+            # calculate gbits and throughput
+            if provider_src == "local":
+                gbits = get_usage_gbits(src)
+            else:
+                gbits = get_usage_gbits(dst)
+            throughput = gbits / 2**30 / request_time
```
Comment (ShishirPatil) [inline skyplane/cli/cli.py:156]: Mention the units in the variable, for eg: `throughput_gbps`

**Review by ShishirPatil — CHANGES_REQUESTED** [review]
This looks good, where we record the throughput and print it out to console. 

What we also want to do is log these metrics to skyplane's logging service like this: `https://github.com/skyplane-project/skyplane/blob/6626ed114e598600aaad8a6d39e6099512462eb8/skyplane/cli/cli.py#L241`
This helps us populate the grid, which can be used by the solver. If you want to do that as a separate PR, let me know I can approve this one. Or if you want to roll it into this PR, that is fine as well.

File: skyplane/cli/common.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 import typer
 from rich.console import Console
+from rich import print as rprint
```
Comment (ShishirPatil) [inline skyplane/cli/common.py:8]: Is there any reason you prefer `rprint` to `console.print` ? It will be good to use console.print for consistency.

Comment (parasj) [reply skyplane/cli/common.py:8]: I use rprint inconsistently in several places throughout the code but also use console in some places. Ideally we should clean it up at some point.

**Review by ShishirPatil — APPROVED** [review]
LGTM

**Pull request merged**